### ISSUE TITLE
Adjusting angle and shape of gauge

### ISSUE THREAD

**Issue by saiyanx** [toorshia/justgage#311]
Have been playing around with justgage and one feature that would be great to see is that we can have more shapes of gauge rather than just donut or semi-circle, I would love to add this feature if someone could give me some pointers on how to achieve it. Thanks for the great work btw!

**Comment by galwaycoder** [comment]
Same, would like to be able to create a gauge similar to that from a car speedometer (somewhere between donut and semi-circle)

**Comment by robertsLando** [comment]
@toorshia  What about this in 2.0?

**Comment by toorshia** [comment]
@robertsLando definitely, there's already similar request: https://github.com/toorshia/justgage/issues/18
